src/agent/onefuzz-agent/src/worker.rs

```diff
@@ -151,6 +214,10 @@ impl State<Ready> {
 
 impl State<Running> {
     pub fn wait(mut self) -> Result<Waited> {
+        while let Ok(res) = self.ctx.from_task_to_agent.try_recv() {
+            info!("received message from server_receiver: {:?}", res);
```

Comment:
Eventually, we plan on doing things with the messages received from the onefuzz-task. For now though, this will be a noop.